.circleci/config.yml

```
@@ -108,6 +109,13 @@ jobs:
             ls ~/package/
             ghr -t ${GITHUB_TOKEN_FOR_RELEASES} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -delete ${VERSION} ~/package/
 
+            echo $DOCKER_HUB_TOKEN | docker login -u thomaseizinger --password-stdin
+
+            docker build -f .docker/Dockerfile . -t comitnetwork/cnd:${VERSION}
```

This should work according the documentation: 
https://docs.docker.com/engine/reference/commandline/build/#tag-an-image--t

```suggestion
            docker build -f .docker/Dockerfile . -t comitnetwork/cnd:${VERSION}  -t comitnetwork/cnd:latest
```

---

Works!